src/akamai/js/akamaiHD_flash.js

```diff
@@ -311,6 +311,21 @@ require("../../../html5-common/js/utils/constants.js");
      * @method OoyalaAkamaiHDFlashVideoWrapper#destroy
      */
     this.destroy = function() {
+      // Pause the video
+      this.pause();
+      
+      // Reset the source
+      this.setVideoUrl('');
+
+      // Pass destroy to flash plugin.
+      this.callToFlash("destroy");
+
+      // Remove the element
+      $('#'+playerId).replaceWith('');
```

Comment:
I just noticed that the variable playerId is used where the wrapper object element is - which was a little confusing.  But since it is that, then removing the element is not as scary as I thought :)